framework/core/js/src/common/Application.tsx

```diff
@@ -248,6 +248,10 @@ export default class Application {
 
   allowUserColorScheme!: boolean;
 
+  refs: Record<string, unknown> = {
```

Comment:
I think we can probably always expect the values to be strings for this one 🤔 

```suggestion
  refs: Record<string, string> = {
```